packages/suite/src/components/suite/Timer.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import formatDuration from 'date-fns/formatDuration';
+import { useLocales } from '@suite-hooks/useLocales';
+
+const formatTimeLeft = (deadline: Date, locale?: Locale) => {
+    const milisecondsLeft = new Date(deadline).getTime() - Date.now();
+    const timeoutDate = new Date();
+
+    timeoutDate.setHours(0, 0, 0, milisecondsLeft > 0 ? milisecondsLeft : 0);
+
+    const duration = {
+        hours: timeoutDate.getHours(),
+        minutes: timeoutDate.getMinutes(),
+    };
+
```

Comment:
better to use [intervalToDuration](https://date-fns.org/v2.29.3/docs/intervalToDuration)

Reply:
553294c65ebc0f98cdd168c80c14542ad398e998